tests/test_sse.py

```diff
@@ -394,3 +394,34 @@ async def func(request):
     )
     assert streamed_data == expected.format(stream_sep)
     await runner.cleanup()
+
+
+@pytest.mark.asyncio
+class TestLastEventId:
+    async def test_success(self, unused_tcp_port, session):
+        async def func(request):
+            async with sse_response(request) as sse:  # type: EventSourceResponse
```

Comment:
Type should be inferred from the return type. If you'd be interested in improving typing, then copy the config from https://github.com/aio-libs/aiohttp-jinja2/blob/master/.mypy.ini
(Comment out any of the options if you want to reduce the size of the initial PR).

```suggestion
            async with sse_response(request) as sse:
```